src/services/constants/stocksEndpoints.js

```diff
@@ -1 +1,2 @@
 export const STOCK_INFO_ENDPOINT = '/stocks?symbol=';
+export const SELL_STOCK_INFO_ENDPOINT = '/stocks/sell?symbol=';
```

Comment:
Suggestion:
I think you can remove `?symbol=` here? you can add this when you call this constant. :)